### PR TITLE
CNV-34765: Add data-test-ids to VM action icons

### PR DESCRIPTION
**Pull request by pcbailey**
## 📝 Description

This PR adds the `data-test-id` attribute to the VM action icons for testing purposes.

Jira issue: https://issues.redhat.com/browse/CNV-34765

### CODE REVIEW

File: src/views/virtualmachines/actions/components/VMActionsIconBar/components/ActionIconButton.tsx [hunk 1]
```diff
@@ -7,7 +7,13 @@ import { VMActionIconDetails } from '@virtualmachines/actions/components/VMActio
```
Comment (metalice) [inline src/views/virtualmachines/actions/components/VMActionsIconBar/components/ActionIconButton.tsx:7]: data test id if very internal, what do u think about passing id? also you can pass id to the button and if needed create a data test id also

Comment (pcbailey) [reply src/views/virtualmachines/actions/components/VMActionsIconBar/components/ActionIconButton.tsx:7]: Guohua asked for data-test-ids, so I think I'll pass id and use that for the data-test-id attribute.

Comment (pcbailey) [reply src/views/virtualmachines/actions/components/VMActionsIconBar/components/ActionIconButton.tsx]: I updated it to use the VM action ids with '-button' as a suffix so I don't have to pass anything at all.

**Pull request merged**